src/components/common/sidebar.js

```diff
@@ -25,30 +27,48 @@ export const Sidebar = ({ pages }) => {
         paper: classes.drawer,
       }}
     >
-      <Box
-        className={classes.toolbar}
-        px={2}
-        display="flex"
-        alignItems="center"
-      >
-        <AirplanemodeActiveIcon className={classes.logo} />
-        <Typography variant="h5">Travelly</Typography>
-      </Box>
-      <Divider />
-      <List>
-        {pages.map(({ title, path, icon }, index) => (
-          <ListItem
-            button
-            key={index}
-            component={NavLink}
-            to={path}
-            activeClassName={classes.activeNavLink}
-          >
-            <ListItemIcon>{icon}</ListItemIcon>
-            <ListItemText>{title}</ListItemText>
-          </ListItem>
-        ))}
-      </List>
+      <div style={{height: "50%"}}>
+        <Box
```

Comment:
Let's move Recent tours content into Recent tours component, not only list but also other staff